### PR TITLE
lint: frontend linting issues

### PR DESCRIPTION
**Pull request by MatriQ**
I occasionally found some front-end lint issues.

### CODE REVIEW

File: web/app/components/app/chat/copy-btn/index.tsx [hunk 1]
```diff
@@ -24,18 +23,18 @@ const CopyBtn = ({
         content={(isCopied ? t('appApi.copied') : t('appApi.copy')) as string}
         className='z-10'
       >
-          <div
```
Comment (MatriQ) [inline web/app/components/app/chat/copy-btn/index.tsx:27]: Alignment issue

File: web/app/components/app/chat/copy-btn/index.tsx [hunk 2]
```diff
@@ -1,10 +1,9 @@
 'use client'
 import React from 'react'
-import Tooltip from '@/app/components/base/tooltip'
```
Comment (MatriQ) [inline web/app/components/app/chat/copy-btn/index.tsx:3]: Import order issue

File: web/app/components/app/text-generate/saved-items/index.tsx [hunk 3]
```diff
@@ -1,17 +1,18 @@
 'use client'
-import React, { FC } from 'react'
```
Comment (MatriQ) [inline web/app/components/app/text-generate/saved-items/index.tsx:2]: Type import issue & import order issue

File: web/app/components/app/text-generate/saved-items/index.tsx [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+import copy from 'copy-to-clipboard'
+import NoData from './no-data'
 import type { SavedMessage } from '@/models/debug'
 import { Markdown } from '@/app/components/base/markdown'
 import { SimpleBtn, copyIcon } from '@/app/components/app/text-generate/item'
-import copy from 'copy-to-clipboard'
 import Toast from '@/app/components/base/toast'
-import NoData from './no-data'
 
-export interface ISavedItemsProps {
```
Comment (MatriQ) [inline web/app/components/app/text-generate/saved-items/index.tsx:12]: We widely export a type fro props, but here is an interface.

File: web/app/components/app/text-generate/saved-items/index.tsx [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
+export type ISavedItemsProps = {
   className?: string
-  list: SavedMessage[],
```
Comment (MatriQ) [inline web/app/components/app/text-generate/saved-items/index.tsx:14]: Comma issue.

File: web/app/components/app/text-generate/saved-items/index.tsx [hunk 6]
```diff
@@ -26,52 +27,54 @@ const SavedItems: FC<ISavedItemsProps> = ({
...
+                    className='space-x-1'
+                    onClick={() => {
+                      copy(answer)
+                      Toast.notify({ type: 'success', message: t('common.actionMsg.copySuccessfully') })
+                    }}>
+                    {copyIcon}
+                    <div>{t('common.operation.copy')}</div>
+                  </SimpleBtn>
 
-                <SimpleBtn
```
Comment (MatriQ) [inline web/app/components/app/text-generate/saved-items/index.tsx:61]: Alignment issue

File: web/app/components/app/text-generate/saved-items/index.tsx [hunk 7]
```diff
@@ -26,52 +27,54 @@ const SavedItems: FC<ISavedItemsProps> = ({
...
   list,
   onRemove,
-  onStartCreateContent
+  onStartCreateContent,
 }) => {
   const { t } = useTranslation()
 
   return (
     <div className={cn(className, 'space-y-3')}>
-      {list.length === 0 ? (
```
Comment (MatriQ) [inline web/app/components/app/text-generate/saved-items/index.tsx:35]: Better line wrapping for ternary operator

**Review by iamjoel — APPROVED** [review]

**Pull request merged**